src/compiler/legend.ts

```diff
@@ -45,7 +44,7 @@ export function compileLegend(model: Model, channel: Channel, def) {
   });
 
   // 2) Add mark property definition groups
-  const props =  isLegend(legend) && legend.properties || {};
+  const props = (typeof legend !== 'boolean' && legend.properties) || {};
```

Comment:
https://stackoverflow.com/questions/899574/which-is-best-to-use-typeof-or-instanceof